spec.md

```diff
@@ -1093,13 +1097,14 @@ If the plugin is unable to complete the NodePublishVolume call successfully, it
 If the conditions defined below are encountered, the plugin MUST return the specified gRPC error code.
 The CO MUST implement the specified error recovery behavior when it encounters the gRPC error code.
 
-Condition | gRPC Code | Description | Recovery Behavior
-| --- | --- | --- | --- |
+| Condition | gRPC Code | Description | Recovery Behavior |
+|-----------|-----------|-------------|-------------------|
 | Volume does not exists | 5 NOT_FOUND | Indicates that a volume corresponding to the specified `volume_id` does not exist. | Caller MUST verify that the `volume_id` is correct and that the volume is accessible and has not been deleted before retrying with exponential back off. |
```

Comment:
NodePublishVolume is also idempotent and should respond `0 OK` if the request is already satisfied by an existing mount.

Reply:
See my comments below. It's slightly different than volume not found case. I'll add some wording at the top about this.